dev/Scroller/Scroller.cpp

```diff
@@ -3612,6 +3612,29 @@ winrt::Rect Scroller::GetDescendantBounds(
         descendantRect.Height });
 }
 
+winrt::AnimationMode Scroller::GetComputedAnimationMode(
+    winrt::AnimationMode const& animationMode)
+{
+    if (animationMode == winrt::AnimationMode::Auto)
+    {
+        bool isAnimationsEnabled;
```

Comment:
```suggestion
        bool isAnimationsEnabled = []()
        {
             auto globalTestHooks = ScrollerTestHooks::GetGlobalTestHooks();

             if (globalTestHooks && globalTestHooks->IsAnimationsEnabledOverride())
             {
                 return globalTestHooks->IsAnimationsEnabledOverride().Value();
             }
             return SharedHelpers::IsAnimationsEnabled();
        }();
```

Reply:
You really like your lambdas. Done.

Reply:
I also really dislike variables initialized to dummy values.

---
In reply to: [293043179](https://github.com/microsoft/microsoft-ui-xaml/pull/850#discussion_r293043179) [](ancestors = 293043179)